consent-ui/src/app/common/to-aspsp/to-aspsp.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+
+@Component({
+  selector: 'consent-app-to-aspsp',
+  templateUrl: './to-aspsp.component.html',
+  styleUrls: ['./to-aspsp.component.scss']
+})
+export class ToAspspComponent implements OnInit {
+  @Input() redirectTo: string;
+  @Input() aspspName: string;
+  @Input() actionName: string;
```

Comment:
It is better to use enum here

Reply:
No need for Enum here because it a common component. Enum is used in pis and ais components